### PR TITLE
Convert GeoConcerns to an Engine

### PR DESCRIPTION
**Pull request by tpendragon**
Converting to an engine required wiping everything and copying
over/renamespacing when appropriate, so the diff here is going to be
monstrous.

Some of the decisions behind what to generate into the app and what sits inside GeoConcerns aren't well thought out, the goal was just to get an Engine running. All of that should be run, thought over, and refactored when appropriate.

### CODE REVIEW

File: tasks/ci.rake [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      ENV['SOLR_TEST_PORT'] = solr.port
+      solr.with_collection(name: 'hydra-test', dir: File.join(File.expand_path('.', File.dirname(__FILE__)), '..', 'solr', 'config')) do
+        FcrepoWrapper.wrap(fcrepo_params) do |fcrepo|
+          ENV['FCREPO_TEST_PORT'] = fcrepo.port
+          while true do
+          end
+        end
+      end
+    end
+  end
```
Comment (eliotjordan) [inline tasks/ci.rake]: The dev_servers and test_servers tasks don't appear in the task list (rake -T).

File: README.md [hunk 2]
```diff
@@ -14,27 +14,23 @@ Rails application for developing Hydra Geo models. Built around Curation Concern
...
-Then:
+## Development
 
-```
-$ rake db:migrate
-$ rake server:development
-```
+1. `bundle install`
+2. `rake engine_cart:generate`
+3. `rake server:development`
```
Comment (eliotjordan) [inline README.md]: Should use `rake dev_servers` instead.

**Comment by eliotjordan** [comment]
There is a problem with navbar styling.
screenshot 2016-04-18 14 43 43

**Comment by eliotjordan** [comment]
'Delete This Raster Work' button is repeated twice.
screenshot 2016-04-18 14 43 24

**Comment by eliotjordan** [comment]
This looks really good @tpendragon. Thanks for doing the heavy lifting/grunt work on this. 🎉  I only saw a handful issues.

**Comment by tpendragon** [comment]
> There is a problem with navbar styling.

This is Curation Concerns.

File: tasks/ci.rake (hunk 1 above)
Comment (tpendragon) [inline tasks/ci.rake]: Not sure why - side effect of them being imported in the Rakefile?

**Comment by tpendragon** [comment]
> 'Delete This Raster Work' button is repeated twice.

Should be fixed.

File: tasks/ci.rake (hunk 1 above)
Comment (tpendragon) [inline tasks/ci.rake]: Ah, nope, they needed descriptions. Fixed.

**Comment by tpendragon** [comment]
@eliotjordan I think your concerns are taken care of. Thanks for catching them.

**Comment by eliotjordan** [comment]
Thanks Trey. As this is a huge change, I'll hold off on merging until others have had a chance to look it over.

**Comment by eliotjordan** [comment]
All right then. This has been rebased and is current up to #124. Any objections to merging this? The sooner the better, so we can start making pull requests against the engine.

**Comment by jrgriffiniii** [comment]
No objections :+1:

**Pull request merged**